docs/source/intervention_models/wasting_treatment/index.rst

```diff
@@ -267,87 +310,91 @@ Congratulate and counsel the mother on appropriate IYCF practices.
 
-For example, a service with treatment coverage of 50%, and a cure-rate of 90% is meeting 50% x 90% = 45% of need (effective coverage of 45%). 
-
 .. image:: effective_coverage_figure.svg
 
-
-| SAM programme treatment coverage: 48.8% (37.4 - 60.4) (this is a point coverage; assumes programmes are not good at case finding) [Isanaka_2021]_
+| SAM programme treatment coverage: 48.8% (37.4-60.4) (this is a point coverage; assumes programmes are not good at case finding) [Isanaka_2021]_
 | MAM programme treatment coverage: same as SAM for now until this website is updated https://acutemalnutrition.org/en/countries 
 
 
-.. note::
-
-  1) Add treatment efficacy for SAM and MAM programmes 
-  2) Discuss how coverage is estimated (SQUEAC surveys). Discuss difference in point-coverage and period-coverage. 
-  3) Update coverage values to 'effective coverage' values
-
+.. todo::
 
+  Discuss how coverage is estimated (SQUEAC surveys). Discuss difference in point-coverage and period-coverage. 
+ 
 
-.. list-table:: Baseline coverage data
-  :widths: 10 10 5 15 20
+.. list-table:: Baseline coverage data (Ethiopia)
+  :widths: 10 10 5 15 20 30
   :header-rows: 1
 
-  * - Location
-    - Subpopulation
-    - Coverage
-    - Value
+  * - Subpop cov
+    - Value (C)
+    - Reference
+    - Value (E)
+    - Reference
     - Note
-  * - Ethiopia
-    - Tx for infants 0-6 months with MAM 
-    - :math:`C_{MAM_{0-6}}`
-    - 48.8% (37.4 - 60.4) for now  
-    - Update this value to post-natal care coverage
-  * - Ethiopia
-    - Tx for infants 0-6 months with any SAM 
-    - :math:`C_{SAM_{0-6}}`
-    - 48.8% (37.4 - 60.4) for now  
-    - Update this value to post-natal care coverage
-  * - Ethiopia
-    - Tx for kids 6-59 months with MAM 
-    - :math:`C_{MAM_{6-59}}`
-    - 48.8% (37.4 - 60.4) ~ stand-in until we find updated data
-    - Coverage of TSFP for kids 6-59 months with MAM
-  * - Ethiopia
-    - Tx for kids 6-59 months with SAM
-    - :math:`C_{SAM_{6-59}}` 
-    - 48.8% (37.4 - 60.4)
-    - Coverage of SC or OTP for SAM kids
+  * - :math:`EC_{SAM_{0-6}}`
+    - 48.8%(95%CI:37.4-60.4) 
+    - [Isanaka_2021]_ 
+    - 70%(95%CI:64-76)
+    - [Zw_2020tx]_
+    - Using the C, E from 6-59mo
+  * - :math:`EC_{SAM_{6-59}}` 
+    - 48.8%(95%CI:37.4-60.4) 
+    - [Isanaka_2021]_ 
+    - 70%(95%CI:64-76)
+    - [Zw_2020tx]_
+    -
+  * - :math:`EC_{MAM_{0-6}}`
+    - 48.8%(95%CI:37.4-60.4) x 
+    - [Isanaka_2021]_ 
+    - 73.1% (95%CI:58.5-87.7) for RUSF
+    - [Ackatia_Armah_2015tx]_
+    - Using the C, E from 6-59mo, *approx 95%CI for E*
+  * - :math:`EC_{MAM_{6-59}}`
+    - 48.8%(95%CI:37.4-60.4)
+    - [Isanaka_2021]_ 
+    - 73.1% (95%CI:58.5-87.7) for RUSF
+    - [Ackatia_Armah_2015tx]_
+    - Update C value for MAM,  *approx 95%CI for E*
+
+
+.. code-block:: python
+
+    #to get the distribution of the above values, use normal distribution with mean and SD 
+
+    SD = (95%UI upper - 95%UI lower) / 2*1.98
```

Comment:
Shouldn't this be 1.96, not 1.98?

Reply:
I think it's for this reason. @aflaxman is that why you suggest 1.98? 

![image](https://user-images.githubusercontent.com/57811075/129972162-e736d703-1dbe-4a51-b08e-f626079f2ad3.png)

Reply:
My bad, I always get confused by this constant.
```
import scipy.stats
scipy.stats.norm().ppf(.975)
1.959963984540054

```